### PR TITLE
fixed multiple liveScreenReader div appending issue

### PR DESCRIPTION
**Pull request by psaha84**
https://coupadev.atlassian.net/browse/CD-171188
Enterprise PR: https://github.com/coupa/coupa_development/pull/70946

- [x] @johnny-lai

### CODE REVIEW

File: coffee/chosen.jquery.coffee [hunk 1]
```diff
@@ -94,7 +94,11 @@ class Chosen extends AbstractChosen
     this.results_build () ->
       @form_field_jq.trigger("chosen:ready", {chosen: this})
 
-    @live_screen_reader = $("<div id='liveForChosenScreenReader' aria-live='polite' class='aria-content'></div>")
+    if $("liveForChosenScreenReader").length  == 0
```
Comment (johnny-lai) [inline coffee/chosen.jquery.coffee:97]: Should `$("liveForChosenScreenReader")` be `$("#liveForChosenScreenReader")`?

Maybe something like?
```
reader = $("#liveForChosenScreenReader")
if reader.length == 0
  @live_screen_reader = $("<div id='liveForChosenScreenReader' aria-live='polite' class='aria-content'></div>")
else
  @live_screen_reader = reader
```

**Comment by psaha84** [comment]
@johnny-lai updated.

**Comment by johnny-lai** [comment]
lgtm

**Comment by johnny-lai** [comment]
[lockit v4.9.3] merged via Lockit CLI on 2020-04-07 12:07:54.293439 -0700 PDT m=+4.425072561

**Pull request merged**